src/debug.ts

```diff
@@ -183,6 +217,24 @@ async function getRuntimeOptions(pkg: VitestPackage) {
   }
 }
 
+function getAttachConfigForBrowser(config: ReturnType<typeof getConfig>, pkg: VitestPackage): vscode.DebugConfiguration {
```

Comment:
I split this out originally with intent to support multiple browsers, but I'm not sure how feasible that is. Vitest only docs playwright with chromium for the vs code/IDE-integrated debugging. Firefox doesn't support inspect-brk flag, and the VS Code integration has some limitations that will make integration challenging - if the browser instance isn't running at attach time, the attach config fails immediately instead of waiting like the built-in chrome attach implementation.

Reply:
I'd say if it doesn't work in Vitest CLI, the extension cannot add support for it, and waiting for it is a bit useless

It's better to throw an error, maybe? Or a warning?